### PR TITLE
Aerospike module

### PR DESCRIPTION
**Pull request by alexshadow007**
It uses http://www.aerospike.com/docs/reference/info to collect metrics.

### CODE REVIEW

**Comment by exekias** [comment]
Jenkins test it

File: metricbeat/module/aerospike/namespace/namespace.go [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+}
+
+// New create a new instance of the MetricSet
+// Part of new is also setting up the configuration by processing additional
+// configuration entries if needed.
+func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
+
+	config := struct{}{}
+
+	logp.Warn("EXPERIMENTAL: The aerospike namespace metricset is experimental")
```
Comment (exekias) [inline metricbeat/module/aerospike/namespace/namespace.go]: We now offer `logp.Experimental`, could you change this to use it?

File: metricbeat/module/aerospike/_meta/docs.asciidoc [hunk 2]
```diff
@@ -0,0 +1,8 @@
+== aerospike Module
+
+This is the aerospike Module. It uses http://www.aerospike.com/docs/reference/info [Info command ] to collect metrics.
```
Comment (exekias) [inline metricbeat/module/aerospike/_meta/docs.asciidoc]: spurious space after `Info command`?

**Comment by alexshadow007** [comment]
@exekias Done

**Comment by exekias** [comment]
Thank you for this awesome contribution @alexshadow007, this is ready to go from my POV, could you add an entry to `CHANGELOG.asciidoc`?

**Comment by alexshadow007** [comment]
@exekias Done

**Comment by tsg** [comment]
@alexshadow007 Do you happen to have a dashboard created for this module? It would be nice to include one as well.

**Pull request merged**